dashboard/app/controllers/projects_controller.rb

```diff
@@ -24,11 +24,12 @@ class ProjectsController < ApplicationController
   def index
   end
 
-  def template
-    # sanitize user input by whitelisting templates we are willing to render
-    head :not_found and return unless TEMPLATES.include? params[:template]
+  def redirect_legacy
```

Comment:
Perhaps worth a short comment either here or in the haml file explaining what legacy means in this context.